### DISCUSSION TITLE
SD Models

### DISCUSSION THREAD

**Discussion by Lucky32167** [lllyasviel/Fooocus#3595] [Q&A]
Hi All,

I wanted to use a LorA which is SD1.5. Unfortunatelly, given that Focus only supports SDXL Checkpoints said LorA didn't yield results even resembling the images it was trained for.

I have two questions:
1. Does anyone know if there are plans to develop Focus for using the SD Checkpoints?
2. Can anyone suggest alternative tool that I could use to work on SD 1.5 LorAs?

Thanks in advance!

**Comment by Yogesh-DevHub** [comment]
The project's readme says "Project Status: Limited Long-Term Support (LTS) with Bug Fixes Only" so, adding SD1.5 will not be happing. 
The alternative can be AUTOMATIC1111/stable-diffusion-webui or Comfy UI.

**Comment by Crocodilian12** [comment]
You _can_ use a 1.5 model as the Refiner, that is, start your generation with an SDXL model . . . and then run the rest of the steps in 1.5. In a case like that, a 1.5 LORA _might_ work (or might not).

Try it and see.

_Can anyone suggest alternative tool that I could use to work on SD 1.5 LorAs?_

-- sure, any of the other UIs will run 1.5. That includes ComfyUI, Webui Forge (same developer as for Fooocus), InvokeAI, probably some others. I'm not sure if Fooocus is unique in being essentially SDXL only, but the other UIs will all work with it.

**Reply by DavidDragonsage** [reply]
Yes, I have tried that technique and it did work. But the results from using the SD1.5 model and the loss of resolution involved made the process essentially worthless. It is better to use an SDXL LoRA if at all possible.

**Reply by Crocodilian12** [reply]
You can experiment with the Refiner switch value -- you can do the switch any any % of steps, if you're trying to get a 1.5 LORA to work, you'd typically want to do the switch earlier.

. . . but in any event, Fooocus is really an SDXL application, now with extension to Pony and in some forks to Flux. I would be surprised if anyone invests much effort in 1.5  . . . and you already have choices in Comfy and Forge for 1.5 which are perfectly good.
